label_studio/projects/functions/next_task.py

```diff
@@ -156,6 +180,11 @@ def get_not_solved_tasks_qs(user, project, prepared_tasks, assigned_flag, queue_
     return not_solved_tasks, user_solved_tasks_array, queue_info
 
 
+def _prioritize_low_agreement_tasks(tasks, lse_project):
+    low_agreement_tasks = tasks.filter(_agreement__lt=lse_project.agreement_threshold, is_labeled=True)
+    return (True, low_agreement_tasks) if low_agreement_tasks.exists() else (False, tasks)
```

Comment:
```suggestion
    return (True, low_agreement_tasks) if low_agreement_tasks else (False, tasks)
```

Reply:
This will produce an additional query and can be reduced to just checking the results of the existing result instead.